op25/gr-op25_repeater/apps/multi_rx.py

```diff
@@ -326,7 +327,7 @@ def close_plots(self):
 
     def toggle_eye_plot(self):
         if 'eye' not in self.sinks:
-            sink = eye_sink_f(plot_name=("Ch:%s" % self.name), chan=self.msgq_id, out_q=self.tb.ui_in_q)
+            sink = eye_sink_f(plot_name=("Ch:%s" % self.name), chan=self.msgq_id, out_q=self.tb.ui_in_q, plot_type=self.tb.http_plot_type)
```

Comment:
It might make more sense to just call `self.tb.http_plot_type` just `self.tb.plot_type` as it is used in areas where `http` might not be the terminal type, and it does default to `image`.